es-data.yaml

```diff
@@ -47,6 +47,11 @@ spec:
         ports:
         - containerPort: 9300
           name: transport
+        lifecycle:
+          preStop:
+            httpGet:
+              path: /_cluster/health?wait_for_status=green&timeout=28800s
+              port: 9300
```

Comment:
shouldn't this be port: 9200?

Reply:
Yes.

Reply:
Without `terminationGracePeriodSeconds: 28800` the `preStop` command will be terminated after 30s even if it's still waiting for the elasticsearch api endpoint to timeout.